tests/Fixtures/Symfony/TestBundle/Controller/ThrowableController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Tests\Fixtures\Symfony\TestBundle\Controller;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+final class ThrowableController
+{
+    /**
+     * @Route(
+     *     methods={"GET"},
+     *     path="/throwable/error"
+     * )
+     */
+    public function error(): void
+    {
+        throw new \Error('Critical failure', 5000);
+    }
+
+    /**
+     * @Route(
+     *     methods={"GET"},
+     *     path="/throwable/exception"
+     * )
+     */
+    public function exception(): void
+    {
+        throw new \RuntimeException('An exception has occurred', 5000);
```

Comment:
@k911 would this be `500` ?

Reply:
I will write feature tests for this special case, but I think yes if you're asking about status code for an error that is never caught.

BTW: If you're looking for the method of handling custom exceptions in symfony you may find this guide useful: https://symfony.com/doc/current/event_dispatcher.html

Reply:
yes I was thinking about code of the exception here being `500` instead of `5000` so it can then be `$e->getCode()`or btw consistent with a server error status code in the logs. Thanks for linking, I did use the event dispatcher quite a lot, is `5000` going to be used as priority for an event then? I might have missed something, apologies if I did.

Reply:
5000 is an example error code defined for testing purposes, it could be 500 but also 100000. You should not use directly exception's codes to status codes anyway this is business logic for the testing bundle you won't be affected by this change. 

Swoole bundle itself after merging this PR will provide the sort of "last resort" exception handler which purpose is to only mitigate swoole's worker failures; not provide business logic for handling exceptions (that will be always on the end-user)

 If you want to, after merging this pr, you could potentially create custom symfony service implementing `K911\Swoole\Server\RequestHandler\ExceptionHandler\ExceptionHandlerInterface` to override logic by default `JsonExceptionHandler` or `ProductionExceptionHandler` which I'm going to provide and support for the above reasons but I'd recommend using `kernel.exception` event listener.

Reply:
sorry my bad didn't see this was part of tests

Reply:
no problem :)